src/header/utils.js

```diff
@@ -64,9 +64,20 @@ export const getToolsMenuItems = ({ studioBaseUrl, courseId, intl }) => ([
     title: intl.formatMessage(messages['header.links.import']),
   },
   {
-    href: `${studioBaseUrl}/export/${courseId}`,
-    title: intl.formatMessage(messages['header.links.export']),
-  }, {
+    href: `${studioBaseUrl}/import/${courseId}`,
+    title: intl.formatMessage(
+      getConfig().ENABLE_TAGGING_TAXONOMY_PAGES === 'true'
+        ? messages['header.links.exportCourse']
+        : messages['header.links.export'],
```

Comment:
Hmm.. I'm not sure we need to do this? But I'm not sure who to ask..

Reply:
I did this to avoid changing the menu for everyone without ENABLE_TAGGING_TAXONOMY_PAGES. But I'm also not sure if we need this. Let's see if we have some comments in the Upstream/CC review.

Reply:
It's fine to change the base menu text ("Export" -> "Export Course"), and I think that's better than having this more complex logic. The "Export Tags" should be gated by the flag though.

Reply:
Done: 20727dccdf5d85fef4f3788050e878218714f7e4